addons/mrp/tests/test_order.py

```diff
@@ -782,9 +782,6 @@ def test_product_produce_6(self):
         mo_form.qty_producing = 3
         mo = mo_form.save()
 
-        mo._post_inventory()
-        self.assertEqual(len(mo.move_raw_ids), 4)
```

Comment:
Does the length of the `move_raw_ids` differ when you don't use `_post_inventory`?

Reply:
yes, it's change. Before `_post_inventory` we have `2` move_raws, and after, `4`.